components/nexus-security/src/main/java/org/apache/shiro/nexus/NexusWebSecurityManager.java

```diff
@@ -28,6 +28,15 @@
 public class NexusWebSecurityManager
     extends DefaultWebSecurityManager
 {
+
+  /**
+   * Constructor which disables all RememberMe functionality.
+   */
+  public NexusWebSecurityManager() {
+    super();
```

Comment:
`super()` is always implied in the default constructor, you never need to make an explicit call